### PR TITLE
Cleanup and add new logs in update service

### PR DESCRIPTION
**Pull request by kavyamani596**

### CODE REVIEW

**Review by chauberahul1993 — APPROVED** [review]

**Review by mohamed-shahid-afrid — APPROVED** [review]

File: svc-update/update/common.go [hunk 1]
```diff
@@ -195,6 +194,7 @@ func (e *ExternalInterface) monitorPluginTask(ctx context.Context, subTaskChanne
 		time.Sleep(time.Second * 5)
 		monitorTaskData.pluginRequest.OID = monitorTaskData.location
 		monitorTaskData.pluginRequest.HTTPMethodType = http.MethodGet
+		l.Log.Debugf("monitor task data payload: %s", string(monitorTaskData.respBody))
```
Comment (amar-shalgar) [inline svc-update/update/common.go]: This needs logfields

File: svc-update/update/simpleUpdate.go [hunk 2]
```diff
@@ -208,12 +209,12 @@ func (e *ExternalInterface) sendRequest(ctx context.Context, uuid, taskID, serve
 			return
 		}
 	}
+	l.Log.Debugf("payload to plugin : %s", updateRequestBody)
```
Comment (amar-shalgar) [inline svc-update/update/simpleUpdate.go]: Log fields are missing

File: svc-update/update/simpleUpdate.go [hunk 3]
```diff
@@ -264,6 +265,7 @@ func (e *ExternalInterface) sendRequest(ctx context.Context, uuid, taskID, serve
 	}
 
 	target.PostBody = []byte(updateRequestBody)
+	l.Log.Debugf("updated payload to plugin : %s", updateRequestBody)
```
Comment (amar-shalgar) [inline svc-update/update/simpleUpdate.go]: log fields missing

File: svc-update/update/startUpdate.go [hunk 4]
```diff
@@ -233,6 +235,7 @@ func (e *ExternalInterface) startRequest(ctx context.Context, uuid, taskID, data
 	}
 
 	target.PostBody = []byte(updateRequestBody)
+	l.Log.Debugf("updated payload to plugin : %s", updateRequestBody)
```
Comment (amar-shalgar) [inline svc-update/update/startUpdate.go]: log fields missing, check cross

**Review by amar-shalgar — CHANGES_REQUESTED** [review]

File: svc-update/umodel/update.go [hunk 5]
```diff
@@ -56,11 +56,13 @@ func GetAllKeysFromTable(table string, dbtype common.DbType) ([]string, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error while trying to get all keys from table - %v: %v", table, err.Error())
 	}
+	l.Log.Debug("all keys from database:", keysArray)
```
Comment (amar-shalgar) [inline svc-update/umodel/update.go]: fields missing

**Review by amar-shalgar — CHANGES_REQUESTED** [review]

**Review by amar-shalgar — APPROVED** [review]

**Pull request merged**